code/build.gradle

```diff
@@ -20,8 +20,8 @@ allprojects {
     repositories {
         mavenCentral()
         google()
-        maven { url "https://jitpack.io"  }
-        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+//        maven { url "https://jitpack.io"  }
+//        maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
```

Comment:
Here, can you just remove the line for jitpack? It's my understanding that we are not using it anymore after the initial release.
You can keep Sonatype uncommented as we'll still want to pull snapshots during development.